compiler/z/codegen/OMRLinkage.cpp

```diff
@@ -1936,7 +1936,7 @@ OMR::Z::Linkage::buildArgs(TR::Node * callNode, TR::RegisterDependencyConditions
    bool isFastJNI, int64_t killMask, TR::Register* &vftReg, bool PassReceiver)
    {
 
-   TR::SystemLinkage * systemLinkage = (TR::SystemLinkage *) self()->cg()->getLinkage(TR_System);
+   TR::SystemLinkage * systemLinkage = (TR::SystemLinkage *) cg()->getLinkage(TR_System);
```

Comment:
This doesn't look correct. I believe the Travis CI build should fail on this. `OMR::Linkage` is definitely an extensible class so calls to `self()` are needed within the object file.